itemadapter/utils.py

```diff
@@ -56,3 +86,37 @@ def is_item(obj: Any) -> bool:
         or is_dataclass_instance(obj)
         or is_attrs_instance(obj)
     )
+
+
+def get_class_field_meta(item_class: type, field_name: str) -> MappingProxyType:
```

Comment:
The name looks good. We can also name it get_field_meta, but no strong preference here.

Reply:
I would keep the `class` there, leave `get_field_meta` for a possible future function that accepts both items and item classes.

Reply:
On second thought, is `get_field_meta_from_class` too verbose?

Reply:
It’s not too verbose to me. And it’s slightly less ambiguous.